cob_people_detection/ros/launch/online_face_recognizer.launch

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+
+<launch>
+ <arg name="camera_namespace" default=""/>
+ <arg name="in_topic_name" value="/face_detector/face_positions"/>
+ <arg name="recognition_method" />
```

Comment:
We need some explanatory comment here, e.g. `<!-- #specifies face recognition algorithm: 0 = Fisherfaces, 1 = Eigenfaces, 2 = LDA2D, 3 = PCA2D, 4 = Deep learning based online recognition (attention: choices 0 and 1 work with a single person in the training data, options 2 and 3 will cause a crash of the program if you do not train at least 2 different people)-->`

Also set a default value, e.g. 2.